### PR TITLE
Bug fixes for the Check duties box.

### PR DESCRIPTION
**Pull request by alexdesi**
### What?
Fix background color, commodity code and wording for the Check duties box.

### Jira link
HOTT-1158

### CODE REVIEW

**Review by jebw — APPROVED** [review]
LGTM

File: spec/helpers/measures_helper_spec.rb [hunk 1]
```diff
@@ -46,16 +46,19 @@
...
 
   describe 'check_how_to_export_goods_link' do
-    subject(:link) { check_how_to_export_goods_link(commodity_code: commodity_code, country_code: '', country_name: '', eu_member: eu_member) }
+    subject(:link) do
+      check_how_to_export_goods_link(declarable: declarable, country_code: '', country_name: '', eu_member: eu_member)
+    end
 
-    let(:eu_member) { true }
+    let(:declarable) do
+      instance_double('Declarable', commodity_code_for_check_duties_service: 'code_for_check_duties', code: 'code')
```
Comment (willfish) [inline spec/helpers/measures_helper_spec.rb]:
```suggestion
      build(:commodity, id: '9801260000')
```

File: spec/helpers/measures_helper_spec.rb [hunk 2]
```diff
@@ -46,16 +46,19 @@
...
+    end
 
     context 'when it is a EU member' do
       let(:eu_member) { true }
-      let(:commodity_code) { '1234567890' }
 
       it 'pass in the code as argument' do
-        expect(link).to include('pc=1234567890')
+        expect(link).to include('pc=code_for_check_duties')
```
Comment (willfish) [inline spec/helpers/measures_helper_spec.rb]:
```suggestion
        expect(link).to include('pc=98012600')
```

**Review by willfish — CHANGES_REQUESTED** [review]
Tiny tweak to integrate rather than mock

**Review by willfish — APPROVED** [review]

File: spec/helpers/measures_helper_spec.rb (hunk 1 above)
Comment (alexdesi) [reply spec/helpers/measures_helper_spec.rb]: 👍

**Pull request merged**